src/main/kotlin/pt/up/fe/ni/website/backend/service/EventService.kt

```diff
@@ -1,14 +1,20 @@
 package pt.up.fe.ni.website.backend.service
 
+import org.springframework.data.repository.findByIdOrNull
 import org.springframework.stereotype.Service
 import pt.up.fe.ni.website.backend.dto.entity.EventDto
 import pt.up.fe.ni.website.backend.model.Event
 import pt.up.fe.ni.website.backend.repository.EventRepository
 
+
+
 @Service
-class EventService(private val repository: EventRepository) {
+class EventService(private val repository: EventRepository, private val accountService: AccountService) {
     fun getAllEvents(): List<Event> = repository.findAll().toList()
 
+    fun getEventById(id: Long): Event = repository.findByIdOrNull(id)
+        ?: throw throw NoSuchElementException(ErrorMessages.eventNotFound(id))
+
     fun createEvent(dto: EventDto): Event {
```

Comment:
Isn't the team members' logic missing when creating events? We should test that too. I think the tests exist for the projects' version

Reply:
For sure, it was an oversight and should be working and being tested properly now